.gitignore

```diff
@@ -42,6 +42,13 @@ tags
 
 ########################################################################
 
+# Mac desktop database
+.DS_Store
+
+# GoLand/PyCharm settings.
+.idea
```

Comment:
Always a good _idea_ 🥁 

Just FYI, I also change my `git config` to include a global .gitignore file:
```
core.excludesfile=/Users/alex/.gitignore_global
```

Reply:
good suggestion, thanks!